online_meetings/report.py

```diff
@@ -1,17 +1,19 @@
 import requests
+from requests.auth import AuthBase
 # standard libraries
 import yaml
 import json
 import logging
 import os
 
-from typing import Dict, Union
+from typing import Dict, Union, List
```

Comment:
I've been using `Sequence` for lists, which it seems like `typing` recommends doing. See [the docs](https://docs.python.org/3/library/typing.html#typing.List`)

Reply:
I'm playing around a bit with `mypy`, it's possible it wants `List` here, so up to your discretion.